### PR TITLE
etcd: Raise default memory limits

### PR DESCRIPTION
**Pull request by NicolasT**
Fixes: #50
See: https://github.com/scality/metalk8s/issues/50

### CODE REVIEW

**Comment by NicolasT** [comment]
@bert-e reset

File: playbooks/group_vars/etcd/10-metal-k8s.yml [hunk 1]
```diff
@@ -6,3 +6,17 @@ docker_dns_servers_strict: False
 # running CentOS 7.4 or higher, with a kernel which contains all required
 # backports.
 docker_storage_options: -s overlay2 --storage-opt overlay2.override_kernel_check=true
+
+# Relax default memory limits for etcd processes.
+# The Kubespray default for this value is 0 (unlimited) when a system has
+# >= 4096MB or memory available, otherwise it's set to 512MB. Our recommended
```
Comment (alxf) [inline playbooks/group_vars/etcd/10-metal-k8s.yml]: i guess you meant: `of memory available` ?

Comment (NicolasT) [reply playbooks/group_vars/etcd/10-metal-k8s.yml]: Yup.

**Comment by NicolasT** [comment]
@bert-e reset

**Review by alxf — APPROVED** [review]

**Pull request merged**